https://github.com/arvidn/libtorrent/commit/f2eefd327f1ca89b053eb1e5f4ed229efe52e47f
https://bugs.gentoo.org/820518

From: Nick Korotysh <kolchaprogrammer@list.ru>
Date: Fri, 25 Jun 2021 22:08:00 +0300
Subject: [PATCH] added missed sources to Python bindings CMakeLists.txt

fixes libtorrent loading module issues:
libtorrent.cpython-39-x86_64-linux-gnu.so: undefined symbol: _Z16bind_sha256_hashv
--- a/bindings/python/CMakeLists.txt
+++ b/bindings/python/CMakeLists.txt
@@ -49,6 +49,7 @@ Python3_add_library(python-libtorrent MODULE WITH_SOABI
 	src/entry.cpp
 	src/error_code.cpp
 	src/fingerprint.cpp
+	src/info_hash.cpp
 	src/ip_filter.cpp
 	src/magnet_uri.cpp
 	src/module.cpp
@@ -56,6 +57,7 @@ Python3_add_library(python-libtorrent MODULE WITH_SOABI
 	src/session.cpp
 	src/session_settings.cpp
 	src/sha1_hash.cpp
+	src/sha256_hash.cpp
 	src/string.cpp
 	src/torrent_handle.cpp
 	src/torrent_info.cpp

